spaces/aliases.json

```
@@ -19,5 +19,7 @@
   "cover": "cover-protocol.eth",
   "iwan": "iwan.eth",
   "oceandao": "officialoceandao.eth",
-  "macaronswap": "macaronswap.eth"
+  "macaronswap": "macaronswap.eth",
+  "mBProtocol" : "m.bsnapshot.eth",
+  "cBProtocol" : "bsnapshot.eth"
```

When creating a space you do not have to edit the `aliases.json` file.
```suggestion
  "macaronswap": "macaronswap.eth"
```

---

But we want to add two aliases, one for mbprotocol and one for cbprotocol

---

You have to create an alias if you had a space that was created before the implementation of ENS and I can't find https://snapshot.org/#/mbprotocol and https://snapshot.org/#/cbprotocol. You are rather on a space creation and you don't need an alias

---

Maybe you want to make a subdomain? https://docs.snapshot.org/spaces/add-custom-domain

---

oh, i think i misunderstood the feature.
We want it to be hosted on snapshot.org, however we don't want it to include `.eth` and potentially other characters as well.
E.g., like `https://snapshot.org/#/dai` doesn't have `.eth`.
Are you saying this is no longer possible?
If it isn't, then we will use our own custom domain.

---

> Are you saying this is no longer possible?
Yes, now the spaces are created with ENS and have the domain extension .eth or .xyz. The solution is to create your own subdomain.

---

I submitted a fix that will only use a domain.
Sorry for the confusion.